### ISSUE TITLE
Input group class dirty issue

### ISSUE THREAD

**Issue by sorxrob** [vuetifyjs/vuetify#163]
So before the 0.8.10 update, everything works fine. But now whenever you put something in the text input, the input-group--dirty isn't applying. You can see it [here](http://imgur.com/a/l74n5).

I haven't played with the vuetify components inside node modules for your information.

**Comment by sorxrob** [comment]
I have to put null values in the all of the object's properties just to make it work.

**Comment by johnleider** [comment]
Please follow the <a href="https://github.com/vuetifyjs/vuetify/blob/master/.github/CONTRIBUTING.md">guidelines</a> on how to report an issue. In particular, provide an example on <a href="https://jsfiddle.net/">www.jsfiddle.net</a> (or a similar service) that reproduces the problem. If necessary, create a repository for us to clone with a minimal reproduction. repositories of actual projects will generally not be accepted .

Thank you.

**Comment by sorxrob** [comment]
Here is my repo https://github.com/sorxrob/vue-test

Just npm install and npm run dev thank you

**Comment by johnleider** [comment]
This is happening because you are binding the v-model of some of your inputs to properties in your data object. For example, the first input Surname references a value of personal_information.surname, but that does not exist. So when you initially update the value, it has to be created and then bound, which is why once you type into another field or select a radio for example, that vue reprocesses the models and makes the input register as dirty.

tldr: don't bind v-model to data values that do not exist.

**Comment by sorxrob** [comment]
Okay thank you. So I should set every property in my object as null?

**Comment by johnleider** [comment]
Yes. Once I did that everything seemed in proper order.
